test.sh

```diff
@@ -661,6 +769,25 @@ function run_other_driver_tests() {
         "$BINDIR"/driver_test filter_test --seed "$SEED"
 }
 
+# #######################################################################
+# Re-run a collection of tests with shared-memory support enabled.
+# We strive to run all the tests that are run in a test execution cycle
+# with shared memory enabled. However, certain test execution configurations
+# may not still be runnable in this mode. So, we will incrementally online
+# remaining tests when they can run successfully in this mode.
+# #######################################################################
+function run_tests_with_shared_memory() {
```

Comment:
OK, here's a **quick-tip**. 

I added a "feature" to `test.sh` in `/main` previously by which you can invoke each of these sub-functions. Here is the help output from `test.sh`:

```
To run a smaller collection of slow running tests,
name the function that drives the test execution.
Examples:
  INCLUDE_SLOW_TESTS=true ./test.sh run_btree_tests
  INCLUDE_SLOW_TESTS=true ./test.sh run_splinter_functionality_tests
  INCLUDE_SLOW_TESTS=true ./test.sh nightly_cache_perf_tests
```

So, if you want to specifically re-run just ALL those tests that exercise shared-memory support, you would do:

`$ INCLUDE_SLOW_TESTS=true ./test.sh run_tests_with_shared_memory`


Greatly improves dev productivity when stabilizing some new code you added.